packages/wallets/src/components/Base/WalletTextField/WalletTextField.tsx

```diff
@@ -57,7 +57,7 @@ const WalletTextField = forwardRef<HTMLInputElement, WalletTextFieldProps>(
                 </div>
                 <div className='wallets-textfield__message-container'>
                     {showMessage && (
-                        <MessageContainer helperMessage={helperMessage} inputValue={value} maxLength={maxLength} />
+                        <HelperMessage helperMessage={helperMessage} inputValue={value} maxLength={maxLength} />
```

Comment:
change the helperMessage props to label or something. Better readability

Reply:
This has already been changed 👍🏼